cmd/installer/helm/cli.go

```diff
@@ -0,0 +1,164 @@
+package helm
```

Comment:
Importing Helm as a Go package is much more difficult (though maybe more controlled) than just shelling out to a `helm` binary, hence this tiny wrapper.